workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb

```diff
@@ -71,7 +71,7 @@
     "from tfx.components import BigQueryExampleGen\n",
```

Comment:
<ul><li>Put this section (along with its title) at the very top of the notebook</li><li>Add learning objectives (see <a href="https://github.com/GoogleCloudPlatform/mlops-on-gcp/blob/master/workshops/kfp-caip-sklearn/lab-01-caip-containers/lab-01.ipynb" target="_blank">this lab</a> to keep formatting consistency)</li><li>"data exploriation" -&gt; "data exploration"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
This PR is to add it InfraValidator to the lab notebook first. I branch from the lab to make the exercises in the format you suggested after this is submitted.

Reply:
Added commit to address data exploration typo. Thanks for catching.